api/ClipboardItem.json

```diff
@@ -246,11 +244,9 @@
             "spec_url": "https://w3c.github.io/clipboard-apis/#optional-data-types",
             "support": {
               "chrome": {
-                "version_added": "121"
-              },
-              "chrome_android": {
-                "version_added": false
+                "version_added": "104"
```

Comment:
Did you mean "124" here?

Reply:
No, I did mean 104, as the bug and commit I listed in the description of the PR suggest.

Reply:
The `supports` static member of this API was introduced in Chrome 121 as determined by the mdn-bcd-collector, so this is triggering a consistency error.  Something's not right here...

Reply:
Ok, so that confirms what I tried to articulate in [this comment](https://github.com/mdn/browser-compat-data/pull/23346#issuecomment-2160584528). That the compat data for the supported types should not be folded as a subfeature under the `supports` feature.